playwright.config.ts

```diff
@@ -1,6 +1,4 @@
-import type { PlaywrightTestConfig } from '@playwright/test';
-
-const config: PlaywrightTestConfig = {
+const config = {
```

Comment:
why this change? we usually recommend a couple of defaults be overridden which give you better performance over time. Thinking about:

```ts
import { defineConfig, devices } from '@playwright/test';

const config = defineConfig({
	fullyParallel: true,
	retries: process.env.CI ? 2 : 0,
	webServer: {
		command: 'pnpm build:svelte && pnpm preview',
		port: 4173,
		timeout: 600_000
	},
	projects: [
		{
      name: 'chromium',
      use: { ...devices['Desktop Chrome'] },
    },
	],
	testDir: 'tests'
});

export default config;
```